src/ts/datasources/raster.ts

```diff
@@ -10,14 +10,13 @@ module powerbi.extensibility.visual.data {
         }
 
         addSources(map, settings) {
-            console.log(settings.raster.tilesize)
             map.addSource('raster', {
                 type: 'raster',
                 tiles: [
                     settings.raster.url
                 ]
                 // tileSize: 256
```

Comment:
@samgehret does tileSize need to be configurable by the end user?  Most WMS sources will generally be 256px outside of Mapbox, but the user may need to configure it.

Reply:
@ryanbaumann Currently, this is handled as a parameter in the URL. Like this https://geodata.state.nj.us/imagerywms/Natural2015?bbox={bbox-epsg-3857}&format=image/png&service=WMS&version=1.1.1&request=GetMap&srs=EPSG:3857&width=256&height=256&layers=Natural2015

Reply:
@samgehret the mapboxgl source component could expose a configurable as well.  Then developers will be able to add whichever tile size their WMS / WMTS source supports.